plugins/civchat2-paper/src/main/java/vg/civcraft/mc/civchat2/commands/GlobalChat.java

```diff
@@ -20,8 +20,6 @@ public class GlobalChat extends BaseCommand {
     @CommandAlias("global|globalchat")
     @Syntax("[message]")
     @Description("Enters global group chat or sends a message to global group chat")
-    @CommandCompletion("@CC_Groups @nothing")
```

Comment:
Is there a reason for removing this?

Reply:
The global chat command has no reason to suggest nl groups to chat in